src/services/File Management/server.js

```diff
@@ -1,9 +1,11 @@
 const express = require("express");
-const fs = require("fs");
+const consume = require("./rmq/rmqConsumerServer");
+const connectToDB = require("./prisma/connectToDB");
+const fileRoutes = require("./routes/filesRoutes");
 require("dotenv").config();
```

Comment:
This line should be the first thing on top of the file as RMQ server needs to get something from the `.env`, so it will fail to get this, and accordingly RMQ server will not run.